pimd/pim_cmd.c

```diff
@@ -3828,7 +3857,7 @@ DEFUN (show_ip_pim_join_vrf_all,
 			first = false;
 		} else
 			vty_out(vty, "VRF: %s\n", vrf->name);
-		pim_show_join(vrf->info, vty, uj);
+		pim_show_join(vrf->info, vty, &sg, uj);
```

Comment:
Shall we skip those vrf that doesn't have pim instance?

Reply:
We already don't crash( as that for every vrf in the system we create a pim_instance and frankly I would like to see the output of that vrf in the show command.

Reply:
That's fine then.